scripts/fast-reboot

```diff
@@ -541,7 +541,10 @@ debug "Stopped radv service..."
 debug "Stopping bgp ..."
 systemctl stop bgp
 debug "Stopped  bgp ..."
-
+# Kill sflow docker	
+debug "Stopping sflow ..."	
+docker kill sflow > /dev/null || true	
```

Comment:
do we need to check if sflow feature is available or not?

Reply:
Agree Guohan and thought about it. However NAT at line 549 was not checking for feature status, hence had the doubt. @jangidrk , could you please address?

Reply:
it looks like it will always return true.

Reply:
looks like 552 lldp example is better.

Reply:
Suggest we follow the LLDP method - "docker kill lldp &> /dev/null || debug "Docker lldp is not running ($?) ...""

Reply:
Thanks. addressed the comments. let me know for NAT also i need to add it here.